Makefile

```diff
@@ -12,12 +12,12 @@ test sure: test-pypackage test-pyapp test-pyramid-app
 
 .PHONY: test-pypackage
 test-pypackage:
-	@bin/make_test pypackage console_script=yes devdata=yes postgres=yes pypi=yes
+	@bin/make_test pypackage console_script=yes devdata=yes postgres=yes __postgres_port=5432 pypi=yes
```

Comment:
We need to hardcode the Postgres port in the tests (instead of letting the cookiecutter randomly generate one) because it needs to match up with the port specified for the Postgres service in the cookiecutter's own `ci.yml` above.